### ISSUE TITLE
VIrus Detected?

### ISSUE THREAD

**Issue by Godly109** [ForgedCore8/OSZsave#4]
![image](https://github.com/user-attachments/assets/fc27e154-2fe4-4618-bd3f-99b291f08587)

**Comment by erius0** [comment]
it is a [known issues](https://github.com/pyinstaller/pyinstaller/issues/6754) with PyInstaller and many other software that packages python scripts into execs and there is no easy way around it - you just have to trust the developers that the provided binary isnt malicious
you could clone this repo and run the script yourself without ever compiling it to an executable file though

**Comment by bytewired9** [comment]
Yeah, cant really do much about it without paying 500$/yr for a cert

**Comment by bytewired9** [comment]
gonna see if i can fix this

**Comment by bytewired9** [comment]
Most recent release is about as good as I can realistically get, given the amount of time & effort im willing to put into this issue
